themes/main/layouts/shortcodes/registration_form.html

```diff
@@ -1,25 +1,14 @@
-<form>
-  <fieldset>
-    <label>
-      {{ T "english_name" }}
-      <input
-        name="name_en"
-        placeholder="{{ T "english_name" }}"
-        autocomplete="english-name"
-      />
-    </label>
-    <label>
-      {{ T "email" }}
-      <input
-        type="email"
-        name="email"
-        placeholder="{{ T "email" }}"
-        autocomplete="email"
-      />
-    </label>
-  </fieldset>
-  <input
-    type="submit"
-    value="{{ T "register" }}"
-  />
-</form>
\ No newline at end of file
+<div id="react-root"></div>
+{{- with resources.Get "js/index.jsx" }}
+  {{- if eq hugo.Environment "development" }}
+    {{- $opts := dict "JSX" "transform"  }}
+    {{- with . | js.Build $opts }}
```

Comment:
**annotation**:
Calling `js.Build` with the `JSX` configuration automatically handles JSX transformation for us. node dependencies are resolved from the `node_modules` directory at the root of the project.